### PR TITLE
Fix several little things in README.md

### PR DESCRIPTION
**Pull request by JohnVeness**
Mostly spelling/grammar. Hope this is OK, thanks :)

### CODE REVIEW

**Comment by JohnVeness** [comment]
Also, I wasn't sure what was meant by "Further SCEE" in line 176 but it reads a little oddly to me.

File: README.md [hunk 1]
```diff
@@ -95,7 +94,7 @@ SCEE has more permissions than StreetComplete: `ACCESS_BACKGROUND_LOCATION` and
     * Allow moving nodes that are part of a way (including a clear warning about changing geometry)
     * Allow disabling and moving the note quest
     * Allow closing notes
-    * Some of the settings below can only be enabled in expert mode
+  * Some of the settings below can only be enabled in expert mode
```
Comment (Helium314) [inline README.md:97]: Hmm, I think this line belongs into the _Expert mode_ section.
Or do you think it's better / more readable with your change?

Comment (JohnVeness) [reply README.md:97]: I see what you mean. When I was first reading the list I forgot I was in the Expert mode section by the time I got to that line, and it seemed like the line was an introduction to the following sections and thus should be less indented. But I don't mind if it's kept as it is now!

Comment (Helium314) [reply README.md:97]: I think both is ok, so it's up to youtr preference.

**Pull request merged**